gigamon/README.md

```diff
@@ -0,0 +1,39 @@
+# Gigamon
+
+## Overview
+[Gigamon][1]  Application Metadata Intelligence empowers your Observability, Security Information and Event Management (SIEM), and Network Performance Monitoring tools with critical metadata attributes across thousands of business, consumer, and IT applications and services. Get deep application visibility to rapidly pinpoint performance bottlenecks, quality issues, and potential network security risks. Gigamon's Application Metadata Intelligence (AMI) helps you monitor and manage complex digital applications for your digital transformation initiatives. This can be achieved through the Gigamon Solution by sending the AMI metadata to Datadog. Some benefits to highlight are Rich Actionable Insights, Boost Security Posture etc.
```

Comment:
Can you include what your integration is specifically monitoring and sending to Datadog?

Reply:
It was already mentioned that it collects the metadata(in json) to datadog.  Please let me know if you are expecting any ?